scripts/inconsistentNugetVersionsInFSharpProjects.fsx

```diff
@@ -12,6 +12,7 @@ open Fsdk
 
 #r "nuget: Microsoft.Build, Version=16.11.0"
 
+#load "../src/FileConventions/Helpers.fs"
```

Comment:
@webwarrior-ws don't forget to rename this .fsx file pls

Reply:
done